bin/crew

```diff
@@ -1205,25 +1205,24 @@ def install_package(pkgdir)
     end
 
     # check if the available rsync command support ACLs
-    rsync_available = File.exist?("#{CREW_PREFIX}/bin/rsync")
-    puts 'rsync is not working. Please install it with \'crew install rsync\''.lightred unless rsync_available
-
-    if Dir.exists? "#{pkgdir}/#{HOME}" then
+      rsync_available = true if `#{CREW_PREFIX}/bin/rsync --version`.include? 'rsync  version'
+      puts 'rsync is not working. Please (re)install it with \'crew remove musl_zstd zstd ; crew install musl_zstd ; crew install rsync\''.lightred unless rsync_available
```

Comment:
Is the indentation off?

Reply:
yes...

Reply:
Is that on purpose or are you planning on fixing it?

Reply:
Done